page-object-code-gen/createPageObjectFromUiDump.kt

```diff
@@ -0,0 +1,132 @@
+import org.w3c.dom.Document
+import java.io.File
+import javax.xml.parsers.DocumentBuilderFactory
+
+/**
+ * inputs:
+ * 1. Path to xml file with UI Dump
+ * output:
+ * Kotlin file with screen code in the same derictory as jar execute
+ */
+fun main(vararg args: String) {
+    val filePath = args.first()
+    if (!File(filePath).isFile) {
+        throw Exception("File is not exist or derictory")
```

Comment:
https://docs.oracle.com/en/java/javase/20/docs/api/java.base/java/io/File.html#isFile()
true if and only if the file denoted by this abstract pathname exists and is a normal file; false otherwise